src/components/header/utils.ts

```
@@ -0,0 +1,63 @@
+    };
+  });
+}
+
+/**
+ * Generates language switch URLs for the header language selector
+ * @param currentPath - Current path without language prefix
+ * @param lang - Current language
+ * @returns Object with language URLs and metadata
+ */
+export function generateLanguageUrls(currentPath: string, lang: "en" | "fr") {
+  if (!currentPath.startsWith("/")) currentPath = `/${currentPath}`;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)